units/en/unit1/dummy-agent-library.mdx

```diff
@@ -4,39 +4,39 @@
 
 This course is framework agnostic because we want to **focus on the concepts of AI agents and avoid getting bogged down in the specifics of a particular framework**. 
 
-Also, we want students to be able to use the concepts they learn in this course in their own projects, in any framework.
+Also, we want students to be able to use the concepts they learn in this course in their own projects, using any framework they like.
 
-Therefore, for this Unit 1, we will use a dummy agent library and simple severless API, which will be our LLM engine. 
+Therefore, for this Unit 1, we will use a dummy agent library and a simple serverless API to access our LLM engine. 
 
-In reality, you would not use these in production, but they will serve as a good **starting point for understanding how agents work**. 
+You probably wouldn't use these in production, but they will serve as a good **starting point for understanding how agents work**. 
 
-But, after this section, you'll **create a simple Agent** using `smolagents`
+After this section, you'll be ready to **create a simple Agent** using `smolagents`
 
 And in the following Units we will also use other AI Agent libraries like `LangGraph`, `LangChain`, and `LlamaIndex`.
 
 To keep things simple we will use a simple Python function as a Tool and Agent. 
 
-We will use built in python packages like `datetime` and `os` so that you can try it out in any environment.
+We will use built-in Python packages like `datetime` and `os` so that you can try it out in any environment.
 
-You can find the notebook [here](https://colab.research.google.com/drive/1MmVo8V9bLkgcVA1RB5LZtX_lO8y_HBxm?usp=sharing). If you **want to run the code**.
+You can follow the process [in this notebook](https://colab.research.google.com/drive/1MmVo8V9bLkgcVA1RB5LZtX_lO8y_HBxm?usp=sharing) and **run the code yourself**.
```

Comment:
The notebook should be stored in a repo, as mentioned in #50. This line can be updated accordingly once it's available.